Kconfig

```diff
@@ -0,0 +1,50 @@
+menu "LoRaWAN Configuration"
+
+    config LORAWAN_SPI_MISO_GPIO
+        int "SPI MISO GPIO"
+        help
+            Set the GPIO number used for SPI MISO.
+
+    config LORAWAN_SPI_MOSI_GPIO
+        int "SPI MOSI GPIO"
+        help
+            Set the GPIO number used for SPI MOSI.
+
+    config LORAWAN_SPI_SCLK_GPIO
+        int "SPI SCLK GPIO"
+        help
+            Set the GPIO number used for SPI SCLK.
+
+    config LORAWAN_SPI_CS_GPIO
+        int "SPI CS GPIO"
+        help
+            Set the GPIO number used for SPI CS.
+
+    config LORAWAN_RESET_GPIO
+        int "SX1262 RESET GPIO"
+        help
+            Set the GPIO number used for RESET.
+
+    config LORAWAN_IRQ_DIO1_GPIO
+        int "SX1262 IRQ (DIO1) GPIO"
+        help
+            Set the GPIO number used for IRQ (DIO1).
+
+    config LORAWAN_BUSY_GPIO
```

Comment:
at least my module has no dedicated BUSY-GPIO as far I can remember - maybe it can be enabled?

Reply:
@bdesterBE if its indeed an optional thing then you should make this config entry depend on another entry that enables it.

Reply:
added define guard to limit busy pin to boards that have one.